### PR TITLE
Update sensor.pvoutput.markdown

### PR DESCRIPTION
**Pull request by Marcel030nl**
**Description:**
Added an example using the template component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/sensor.pvoutput.markdown [hunk 1]
```diff
@@ -35,3 +35,30 @@ Configuration variables:
...
+```yaml
+sensor:
+  - platform: pvoutput
+    system_id: YOUR_SYSTEM_ID
+    api_key: YOUR_API_KEY
+    scan_interval: 150
+  - platform: template
+    sensors:
+      power_consumption:
+        value_template: '{% if is_state_attr("sensor.pvoutput", "power_consumption", "NaN") %}0{% else %}{{ states.sensor.pvoutput.attributes.power_consumption }}{% endif %}'
```
Comment (fabaff) [inline source/_components/sensor.pvoutput.markdown]: All templates needs to be wrapped with `{% raw %}...{% endraw %}` otherwise they are rendered.

**Comment by Landrash** [comment]
Wouldn't this be better suited in the [Example/Cookbook](https://home-assistant.io/cookbook/) section since it's a good generalized example also?

**Comment by fabaff** [comment]
I though about that too but for me it looks like that the template stuff it's very sensor-specific and could help users of the PVOutput sensor to find all details at one place.

**Comment by Landrash** [comment]
Good work. Merging this in but feel free to make a copy or similar in the Cookbook section.

**Pull request merged**